src/OneDWaveEquation.jl

```diff
@@ -85,11 +85,12 @@ function WaveEquation(c, grid)
     return FourierFlows.Equation(L, calcN!, grid)
 end
 
-struct Vars{A, B} <: AbstractVars
-     ξ :: A
-     u :: A
-    ξh :: B
-    uh :: B
+struct Vars{A, B, A2} <: AbstractVars
+          ξ :: A
+          u :: A
+         ξh :: B
+         uh :: B
+  ξ0u0_mean :: A2
```

Comment:
Is it necessary to store these in one variable?

Can make the `struct` mutable with variables `Ξ` and `U`?

Or, `mean_ξ` and `mean_u`.

Reply:
done